motionSetPointsApp/Db/motionSetPointsDoubleAxis.substitutions

```diff
@@ -17,13 +17,15 @@ file $(UTILITIES)/db/unit_setter.template {
 
     { "\$(AXIS0).EGU", "\$(P)COORD0:NO_OFF", "" }
     { "\$(AXIS0).EGU", "\$(P)COORD0:OFFSET", "" }
+    { "\$(AXIS0).EGU", "\$(P)COORD0:OFFSET:SP", "" }
```

Comment:
Do we need to pass "NPP" as third argument, or is it OK to initiate processing from this unit propagation?

Reply:
yes, I think using NPP would be safest in this case

Reply:
All of the others could probably have NPP explicitly set I think?

Reply:
It depends what you want to achieve - if you use NPP then any CA monitor that was attached before you change units will not notice the change. The NPP is likely important on an output record an you don't want to e.g. accidentally send a position. It is usually not important on a input record, but if the input record has a FLNK then it may trigger something you don't want to happen. So NPP everywhere is safer, but may not have the desired effect of propagating a units change to the GUI if it happens after the GUI has connected.

Reply:
I think given that .EGU on the motor record does not change frequently we are probably OK - They will be updated on next IOC restart at the latest? I don't think there's any logic that relies on units being updated as quickly as possible.